app/core/ballot.py

```diff
@@ -28,8 +28,8 @@ def get_ballot(
     except Exception as error:
         print(sys.exc_info())
         raise HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            detail="get ballot failed",
+            status_code=status.HTTP_404_NOT_FOUND,
```

Comment:
These 404 changes feels like a separate PR.

Reply:
A new issue was added to cover this.

Reply:
new issue was written for these